README.md

```diff
@@ -5,88 +5,112 @@
-       implementation("com.github.klaviyo.klaviyo-android-sdk:push-fcm:2.0.0")
-   }
-   ```
-   ```groovy
-   // build.gradle
-   dependencies {
-       implementation "com.github.klaviyo.klaviyo-android-sdk:analytics:2.0.0"
-       implementation "com.github.klaviyo.klaviyo-android-sdk:push-fcm:2.0.0"
-   }
-   ```
-
-## Analytics SDK
-
-### Configuration
-
-The SDK must be configured with the public API key for your Klaviyo account.
-We require access to the `applicationContext` so the SDK can be responsive to
-changes in network conditions and persist data with `SharedPreferences`.
-You must also register the Klaviyo SDK for activity lifecycle callbacks per the example code:
+      ```
+   </details>
+
+## Initialization
+The SDK must be initialized with the 6-character 
+[public API key](https://help.klaviyo.com/hc/en-us/articles/115005062267#difference-between-public-and-private-api-keys1)
+for your Klaviyo account. We require access to the `applicationContext` so the
+SDK can be responsive to changes in network conditions and persist data via
+`SharedPreferences`. You must also register the Klaviyo SDK for activity lifecycle
+callbacks per the example code, so we can gracefully manage background processes.
 
 ```kotlin
+// Application subclass 
 import android.app.Application
 import com.klaviyo.analytics.Klaviyo
-import com.klaviyo.analytics.lifecycle.KlaviyoLifecycleMonitor
-import com.klaviyo.push.KlaviyoPushService
 
 class TestApp : Application() {
     override fun onCreate() {
         super.onCreate()
 
+        /* ... */
+        
+        // Initialize is required to use any Klaviyo SDK functionality 
         Klaviyo.initialize("KLAVIYO_PUBLIC_API_KEY", applicationContext)
 
+        // Required for the SDK to properly respond to lifecycle changes such as app backgrounding 
         registerActivityLifecycleCallbacks(Klaviyo.lifecycleCallbacks)
     }
 }
 ```
-`Klaviyo.initialize()` *must* be called before any other SDK methods can be invoked, thus it 
-should be added to your `Application.onCreate`.
 
-### Identifying a Profile
+`Klaviyo.initialize()` **must** be called before any other SDK methods can be invoked.
+Because we require lifecycle callbacks, it is necessary to subclass
+[`Application`](https://developer.android.com/reference/android/app/Application)
+to initialize and register callbacks in `Application.onCreate`.
 
-The SDK provides helpers for identifying profiles and syncing via the
-[Klaviyo client API](https://developers.klaviyo.com/en/reference/create_client_profile).
+## Profile Identification
+The SDK provides methods to identify profiles via the
+[Create Client Profile API](https://developers.klaviyo.com/en/reference/create_client_profile).
 All profile identifiers (email, phone, external ID, anonymous ID) are persisted to local storage
 so that the SDK can keep track of the current profile.
 
 Klaviyo SDK does not validate email address or phone number inputs locally, see
```

Comment:
reword the start here

Reply:
Sure. We can also update this section whenever we implement this ticket: https://klaviyo.atlassian.net/browse/CHNL-3990 to be more explicit about what we do when identifiers are invalid according to the server.